app/models/spree/variant_decorator.rb

```diff
@@ -60,14 +60,14 @@ def google_base_item_color
     end
 
     private
+
     def warehouse_count_on_hand
-      warehouse_id = stock_locations.find_by(name: "Warehouse").id
-      for stock_item in stock_items
-        if stock_item.stock_location_id == warehouse_id
-          return stock_item.count_on_hand
-        end
-      end
+      stock_items.where(stock_location_id: current_warehouse_id).map(&:count_on_hand).sum
     end
 
+    def current_warehouse_id
```

Comment:
are we sure we want this feed to be based off the US warehouse?

Reply:
For now I think that's the plan. But long term it feels like we are in the dark.